tests/integration/models.py

```diff
@@ -48,3 +48,18 @@ class Meta:
     redirect_to_path = factory.LazyFunction(lambda: url_for("platform.list_grants"))
     expires_at_utc = factory.LazyFunction(lambda: datetime.datetime.now() + datetime.timedelta(minutes=15))
     claimed_at_utc = None
+
+
+class _CollectionSchemaFactory(factory.alchemy.SQLAlchemyModelFactory):
+    class Meta:
+        model = CollectionSchema
+        sqlalchemy_session_factory = lambda: db.session  # noqa: E731
+
+    id = factory.LazyFunction(uuid4)
+    name = factory.Sequence(lambda n: "Collection %d" % n)
+
+    created_by_id = factory.LazyAttribute(lambda o: "o.user.id")
```

Comment:
I can see you've copied this from above, but I clearly made a mistake as this should be `o.created_by.id`. I'd expect this to error but we're actually just returning a string, not referencing attributes on the object via `o`. Can you fix this and the magiclink above please? 🙏 

```suggestion
    created_by_id = factory.LazyAttribute(lambda o: o.created_by.id)
```